app/views/applicant_details/show.html.erb

```diff
@@ -11,7 +11,7 @@
       <ul>
         <li>Gender: <%= gender_description(@applicant_detail.gender) %></li>
         <li><%= @us_citizen %></li>
-        <li>Demographic: <%= demographic_description(@applicant_detail) %></li>
+        <li>Demographic: <%= @applicant_detail.demographic_name %><%= " - #{@applicant_detail.demographic_other}" if @applicant_detail.demographic_other.present? && demographic_description(@applicant_detail) == "Other" %></li>
       </ul>
```

Comment:
If demographic_name in the Demographic model is replaced with
```
def demographic_name
    if self.demographic_other.present?
      Demographic.find(self.demographic).name + " - " + self.demographic_other
    else
      Demographic.find(self.demographic).name
    end
  end
```
(see my comment for admin/demographic.rb)
then lines 14 can be replaced with:
```
<li>Demographic: <%= @applicant_detail.demographic_name %></li>
```
And in this case we don't need DemographicsHelper